### PR TITLE
rt rust project

### PR DESCRIPTION
**Pull request by OGordoo**

### CODE REVIEW

File: subjects/rt/README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+As your ray tracer will probably be a bit slow to render high resolution scenes, you should make 4 .ppm images for the auditers to evaluate. The scenarios of these 4 images that you have to create consist of:
+
+- a scene with a sphere;
+- a scene with a plane and a cube with lower brightness than in the sphere image;
+- a scene with one of each of all the objects (one cube, one sphere, one cylinder and one plane);
+- a scene like the previous one, but with the camera in another position (thus generating the same image from a different perspective).
+
+All of the images should be in the format of 800x600. But while testing you can use smaller resolution images in order to not wait for a long time.
+
+Another aspect you should consider is that the auditer will have to use your ray tracer, so make it as usable and optimized as possible.
```
Comment (LEEDASILVA) [inline subjects/rt/README.md:37]: Maybe we can ask them to create a small documentation on how to use their ray tracing API. You can give some points like in the `mini-framework` project, and also add to the audits

File: subjects/rt/audit/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+###### Did the student include a README for the project explaining how to use the ray tracer?
+
+###### Is it possible to add textures to the surface of the objects?
+
+###### Is it possible to make reflective and/or refractive objects?
+
+###### Is it possible to add particles?
+
+###### Is it possible to add fluids?
```
Comment (LEEDASILVA) [inline subjects/rt/audit/README.md:35]: for the bonus you are missing the `+` in front of the question

File: subjects/rt/audit/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+#### Functional
+
+##### Using the ray tracer construct any scene you want, including at least one of all objects. (this can take a while to render, so in the meanwhile you can answer the next questions)
+
+###### Does the image corresponds to the scene you created?
+
```
Comment (LEEDASILVA) [inline subjects/rt/audit/README.md:6]: maybe you can add a question that asks the auditor to try and create one of the same scenes given to them (the 4 .ppm images). This way we could make sure that the images where really made using their program. what do you think?

**Review by LEEDASILVA — CHANGES_REQUESTED** [review]

File: subjects/rt/audit/README.md (hunk 3 above)
Comment (OGordoo) [reply subjects/rt/audit/README.md:6]: it can  be difficult to recreate the same image, but if they already have the code to do it yeah, it is possible

Comment (LEEDASILVA) [reply subjects/rt/audit/README.md:6]: ok, so it will be better if we leave it like you already have

File: subjects/rt/README.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
+## rt
+
+There are two ways to render a 3d scene into a 2d image: `rasterization` that basically converts the shapes and geometric to pixels and applies calculations to obtain the color, shadows, refraction, etc... of those pixels. The other method is called `ray tracing` and consists in drawing each pixel with already its color, shadow, refraction, reflection, etc....
+
+Here we can see a visual difference between each method:
+
+![image.png](raytrace.png)
+
+![image.png](raytrace2.png)
```
Comment (augusto-mantilla) [inline subjects/rt/README.md:9]: I think now you should delete those two. The other one looks much better anyway haha

**Review by augusto-mantilla — CHANGES_REQUESTED** [review]

File: subjects/rt/README.md (hunk 4 above)
Comment (OGordoo) [reply subjects/rt/README.md:9]: ahah thanks, done

**Review by augusto-mantilla — APPROVED** [review]

**Comment by augusto-mantilla** [comment]
We wait for @Frenchris review to merge

**Comment by xpetit** [comment]
@OGordoo 
Can you make the image smaller ?
My favorite sites to do so :
https://kraken.io/web-interface
https://squoosh.app/
https://compress-or-die.com/

**Comment by xpetit** [comment]
@OGordoo not avif, this is too edgy ! png or jpeg please

**Review by LEEDASILVA — APPROVED** [review]

**Comment by Frenchris** [comment]
Checked with afonso.

**Pull request merged**